libvast/src/io/save.cpp

```diff
@@ -0,0 +1,37 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/io/save.hpp"
+
+#include "vast/error.hpp"
+#include "vast/io/write.hpp"
+#include "vast/path.hpp"
+
+#include <cstdio>
+
+namespace vast::io {
+
+caf::error save(const path& filename, span<const byte> xs) {
```

Comment:
```suggestion
[[nodiscard]] caf::error save(const path& filename, span<const byte> xs) {
```

Reply:
The [[nodiscard]] needs to only be at the declaration.